ingestJSONFiles/UofM_One_PD.json

```diff
@@ -0,0 +1,24 @@
+{
+    "data": [
+        {
+            "Author(s)": "Arnold, Matthew",
+            "ISBN": "",
+            "OCLC": "183163",
+            "File ID 1": "39015091349970",
+            "Title": "Democratic Education",
+            "Volume / Edition": "",
+            "Pub Date": "1962",
+            "Rights status linked record": "Publisher-controlled,Public Domain,Cleared by NYPL rights team",
+            "Rights status (from Rights status linked record)": "publisher-controlled,public domain,cleared by rights team",
+            "Rights classification by publisher": "A",
+            "Notes on rights, rights records": "",
+            "Access in DRB": "Full Access",
+            "Digitial copy at press?": "yes",
+            "Digital copy received": "checked",
+            "Project steps": "QA test,On hold",
+            "Next step (from Project steps)": "QA test,On hold",
+            "Date of ingest": "",
+            "Status updated on": "4/8/2024"
+        }
+    ]
+}
```

Comment:
what's our style guidelines for new lines after end of files?

Reply:
We don't have style guidelines for new lines after end of files to my knowledge, but I would suggest matching the indentation of the other lines of the file like in this json file.